packages/react/src/utils/queryKeys.ts

```diff
@@ -1,18 +1,61 @@
+import type { QueryKey } from '@tanstack/react-query';
+import type { BytesLike } from 'fuels';
+
 export const QUERY_KEYS = {
-  account: 'account',
-  accounts: 'accounts',
-  assets: 'assets',
-  chain: 'chain',
-  isConnected: 'isConnected',
-  provider: 'provider',
-  balance: 'balance',
-  wallet: 'wallet',
-  transaction: 'transaction',
-  transactionReceipts: 'transactionReceipts',
-  nodeInfo: 'nodeInfo',
-  connectorList: 'connectorList',
-  currentConnector: 'currentConnector',
-  currentNetwork: 'currentNetwork',
+  base: ['fuel'] as QueryKey,
+  account: (): QueryKey => {
+    return [...QUERY_KEYS.base, 'account'];
```

Comment:
we could use also `QUERY_KEYS.base.concat('accounts')` as it creates a new array.

Reply:
nice catch, `concat` is also way faster than the spread operator. I'm fixing it.

Reply:
https://github.com/FuelLabs/fuels-npm-packs/pull/60/commits/5487d4704e76bb33184ba8bd3e1b424f0b45c3da